packages/publish/src/lib/pack-directory.ts

```diff
@@ -69,6 +73,47 @@ export async function packDirectory(_pkg: Package, dir: string, options: PackCon
   );
 }
 
+export const PUBLISH_CONFIG_FIELDS = [
+  'bin',
+  'main',
+  'exports',
+  'types',
+  'typings',
+  'module',
+  'browser',
+  'esnext',
+  'es2015',
+  'unpkg',
+  'umd:main',
+  'cpu',
+  'os',
+];
+
+async function overwritePublishConfig(manifestLocation: string) {
```

Comment:
I would prefer "override" instead of "overwrite" because the latter can be confused by thinking it's overwrites the entire file which is far from it. Also, pnpm wrote the something similar in their docs

> It is possible to override some fields in the manifest before the package is packed. The following fields may be overridden: